backend/src/models/service.ts

```diff
@@ -94,6 +94,20 @@ export class Service extends BaseEntity {
   })
   discoveredBy: Scan;
 
+  /** Scan that discovered this product/cpe (wappalyzer, intrigue-ident, censys). */
+  @Column({
+    nullable: true,
+    type: 'text'
+  })
+  productSource: string | null;
```

Comment:
either put this under `products` or let's remove this in this PR for now

Reply:
Removing from the PR for now